libfwupdplugin/fu-efivar-freebsd.c

```diff
@@ -55,7 +56,7 @@ fu_efivar_delete_with_glob_impl (const gchar *guid, const gchar *name_glob, GErr
 	efi_str_to_guid (guid, &guid_to_delete);
 
 	while (efi_get_next_variable_name (&guidt, &name)) {
-		if (memcmp (&guid_to_delete, guidt, sizeof (guidt)) != 0)
+		if (memcmp (&guid_to_delete, guidt, sizeof (guid_to_delete)) != 0)
```

Comment:
isn't this the same?

Reply:
No, `guidt` is a pointer, while `guid_to_delete` isn't.

Reply:
Ack, gotcha, thanks.